project/plugins.sbt

```diff
@@ -1,7 +1,7 @@
 addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.12.1")
 addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.11.0")
 addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "3.10.0")
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.2-1")
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "2.1.1")
```

Comment:
may need some testing
- some breaking changes in https://github.com/sbt/sbt-pgp/releases/tag/v2.0.0
- was reverted once in b5176f01ff5e1b6cab7d7896c44002e5dd0a6c26